bpf/lib/icmp6.h

```diff
@@ -1,7 +1,7 @@
 /* SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause) */
 /* Copyright Authors of Cilium */
 
-#if !defined(__LIB_ICMP6__) && defined(ENABLE_IPV6)
+#ifndef __LIB_ICMP6__
```

Comment:
Nit: change this `#ifndef` and subsequent `#define` to `#pragma once` for simplicity.

Reply:
Humm.. there is a mix of `#pragma once` and `#ifndef` as guards in `lib/`, and in general across the codebase. I am personally biased towards `#ifndef` (PTS?). Is there a consensus on this?

Reply:
@ti-mo :arrow_up: ?